devstack/lib/nfp

```diff
@@ -123,10 +123,6 @@ function create_ep_and_nsp {
  --shared True\
  default
 
-    gbp ep-create\
- --external-segments default\
- ext_connect
```

Comment:
Having one external policy(ep) does not let us create another ep with the consuming rule sets while launching the chain, in such case any consuming rule sets  will not be applied. So it is of no use creating an empty ep here.